src/style/variables.scss

```diff
@@ -1,4 +1,21 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
+
 $iui-border-radius: 3px;
 $iui-blur-filter: blur(5px);
+
+
+// All inputs
+$iui-component-offset: 3px;
+
+$iui-component-height: $iui-baseline * 4 - $iui-component-offset * 2; // 38px
+$iui-component-height-small: $iui-baseline * 3 - $iui-component-offset * 2; // 27px
+$iui-component-height-large: $iui-baseline * 5 - $iui-component-offset * 2; // 49px
+
+$iui-component-padding-width: $iui-sm; // 12px
```

Comment:
Maybe replace `width` and `height` to `horizontal` and vertical` because it really looks odd when I see width and padding in the same variable.

Reply:
Done